### PR TITLE
updating API proxy images

### PR DESCRIPTION
**Pull request by huguesBouvier**
updating API proxy images

### CODE REVIEW

File: edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 ARG DIR=.
 
-FROM alpine:3.12.0
+FROM alpine:3.13.1
```
Comment (yophilav) [inline edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile:8]: We are still using 3.12 alpine in EdgeHub & EdgeAgent.

**Comment by yophilav** [comment]
Please also merge this change in to release1.2 as well

File: edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile (hunk 1 above)
Comment (huguesBouvier) [reply edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile:8]: Varun told me it is generating errors so I am updating it

**Comment by huguesBouvier** [comment]
> Please also merge this change in to release1.2 as well

yup

**Review by and-rewsmith — APPROVED** [review]
LGTM

**Pull request merged**